### PR TITLE
update: Bump Angular to V12

### LINKED ISSUE #75: add  angular 13 support
Since angular 13 now works with full ivy enable, it would be nice to have it enabled on this library.

Thx a lot!

### PR DESCRIPTION
**Pull request by Heatmanofurioso**
Partially closes #75 
Will perform another PR to bump to V13 is the migration goes well.

The automatic Angular `ng update @angular/core@12` was performed,  npm run test:dev``  were ran and I tested the lib using a project that uses it as a dependency. 
Everything seemed to run fine.

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -46,12 +51,12 @@
...
               "sourceMap": false,
               "namedChunks": false,
-              "aot": true,
               "extractLicenses": true,
               "vendorChunk": false,
               "buildOptimizer": true
             }
-          }
+          },
+          "defaultConfiguration": ""
```
Comment (hakimio) [inline angular.json:59]: Might also be a good idea to run optional "production-by-default" migration:
`ng update @angular/cli@12 --migrate-only production-by-default`

**Comment by hakimio** [comment]
Version number in `projects/ngx-google-analytics/package.json` might need to be incremented as well.

File: angular.json (hunk 1 above)
Comment (Heatmanofurioso) [reply angular.json]: Thank you for the notice. Was so used to not having the flag anymore, I didn't notice it was here.
Fixed

**Comment by Heatmanofurioso** [comment]
Do let me know if there's anything blocking you merging and deploying this, that I can fix or improve
Unless it's just lack of time :)

**Review by maxandriani — APPROVED** [review]

**Pull request merged**